chapters/3-package-information.md

```diff
@@ -971,41 +1179,51 @@ where:
 
 Example:
 
-    ExternalRef: SECURITY cpe23Type cpe:2.3:a:pivotal_software:spring_framework:4.1.0:*:*:*:*:*:*:*
+```text
+ExternalRef: SECURITY cpe23Type cpe:2.3:a:pivotal_software:spring_framework:4.1.0:*:*:*:*:*:*:*
+```
 
-    ExternalRef: PERSISTENT-ID swh swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2
+```text
+ExternalRef: PERSISTENT-ID swh swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2
+```
 
-    ExternalRef: OTHER LocationRef-acmeforge acmecorp/acmenator/4.1.3-alpha
+```text
+ExternalRef: OTHER LocationRef-acmeforge acmecorp/acmenator/4.1.3-alpha
+```
 
 **3.21.6** RDF: property `externalRef` in class `spdx:Package` of type `spdx:ExternalRef` 
 
 Example (for a listed location):
 
-    <spdx:Package  rdf:about="...">
-        ...
-        <spdx:externalRef>
-            <spdx:ExternalRef>
-                <spdx:referenceCategory rdf:resource="spdx:referenceCategory_packageManager" />
-                <spdx:referenceType rdf:resource="http://spdx.org/rdf/refeferences/maven-central" />
-                <spdx:referenceLocator>org.apache.commons:commons-lang:3.2.1</spdx:referenceLocator>
-            </spdx:ExternalRef>
-        </spdx:externalRef>
-        ...
-    </spdx:package>
+```text
+<spdx:Package  rdf:about="...">
```

Comment:
looks like there's an extra empty space between `Package` and `rdf`. I don't particularly care but I'm noting it as I go through these edits  :)